test/testcondition.cpp

```diff
@@ -1167,6 +1168,20 @@ class TestCondition : public TestFixture {
         ASSERT_EQUALS("[test.cpp:3]: (warning) Logical conjunction always evaluates to false: t == 0 && t == 1.\n", errout.str());
     }
 
+    void incorrectLogicOperator11() {
+        check("void foo(int i, const int n) { if ( i < n && i == n ) {} }");
+        ASSERT_EQUALS("[test.cpp:1]: (warning) Logical conjunction always evaluates to false: i < n && i == n.\n", errout.str());
```

Comment:
As far as I know a "logical conjunction" means that the conditions are overlapping and a "logical disjunction" means that the conditions are opposite. I would say this is a "disjunction". Do you disagree?

Would it break many existing tests to change this word?